mqtt/src/test/java/akka/stream/alpakka/mqtt/javadsl/MqttSourceTest.java

```diff
@@ -252,7 +253,7 @@ public void supportWillMessage() throws Exception {
     MqttSourceSettings settings1 = MqttSourceSettings.create(
       sourceSettings
         .withClientId("source-spec/testator")
-        .withBroker("tcp://localhost:1337")
+        .withBroker("tcp://localhost:" + String.valueOf(proxyPort))
```

Comment:
This would be a bit cleaner as `s"tcp://localhost:$proxyPort"`

Reply:
Yes, but Java does not do that yet, AFAIK ;-)

Reply:
Oh, my bad! Didn't realize it was the java test.

LGTM, anyway :)